lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1065,6 +1083,19 @@ def loki_push_api(self):
         """Fetch Loki Push API endpoint sent from LokiPushApiProvider through relation data.
 
         Returns:
-            Loki Push API endpoint
+            The Loki Push API endpoint (or None) if the relation this `LokiPushApiConsumer` tracks
+            has limit 1; with any other relation limit, it returns a list of Loki Push API
+            endpoints, which may be empty in case there are no relation instances.
```

Comment:
Add return type hint?
Seems like it's `Union[None, str, List[str]]`.
Would it make sense to have a return type of `Optional[List[str]]` or maybe even `List[str]`?

Reply:
Following in this issue #49